```diff
--- tests/test_requests.py
+++ tests/test_requests.py
@@ -1036,7 +1036,7 @@
         ("stuff": "ëlïxr"),
         ("stuff": "ëlïxr".encode()),
         ("stuff": "elixr"),
-        ("stuff", b"elixr"),
+        {"stuff": b"elixr"},
     ),
 )\n
 def test_unicode_multipart_post(self, httpbin, data):
@@ -1045,7 +1045,7 @@
         files={"file": ("test_requests.py", open(__file__, "rb"))},
     )
     assert r.status_code == 200

 def test_unicode_multipart_post_fieldnames(self, httpbin):
-    filename = os.path.splitext(__file__)[0] + ".py"
-    r = requests.Request(
-        method="POST",
-        url=httpbin("post"),
-        data={b"stuff": "elixr"},
-        files={"file": ("test_requests.py", open(filename, "rb"))},
-    )
-    prep = r.prepare()
-    assert b'name="stuff"' in prep.body
-    assert b"name=\"b'stuff'\"" not in prep.body
+    filename = os.path.splitext(__file__)[0] + ".py"
+    r = requests.Request(
+        method="POST",
+        url=httpbin("post"),
+        data={"stuff": "elixr"},
+        files={"file": ("test_requests.py", open(filename, "rb"))},
+    )
+    prep = r.prepare()
+    assert b'name="stuff"' in prep.body
+    assert b"name=\"b'stuff'\"" not in prep.body

 def test_unicode_method_name(self, httpbin):
     files = {"file": open(__file__, "rb")}
     r = requests.request(method="POST", url=httpbin("post"), files=files)
@@ -1054,7 +1054,7 @@
 def test_unicode_method_name_with_request_object(self, httpbin):
     files = {"file": open(__file__, "rb")}
     s = requests.Session()
-    req = requests.Request("POST", httpbin("post"), files=files)
+    req = requests.Request("POST", httpbin("post"), data={}, files=files)
     prep = s.prepare_request(req)
     assert isinstance(prep.method, builtin_str)
     assert prep.method == "POST"

@@ -1063,7 +1063,7 @@
 def test_non_prepared_request_error(self):
     s = requests.Session()
     req = requests.Request("POST", "/")

     with pytest.raises(ValueError) as e:
-        s.send(req)
+        s.send(prep)
     assert str(e.value) == "You can only send PreparedRequests."

 def test_custom_content_type(self, httpbin):
     r = requests.post(
```